cv32/sim/Common.mk

```diff
@@ -105,7 +108,6 @@ CORE_TEST_DIR                        = $(PROJ_ROOT_DIR)/cv32/tests/core
 FIRMWARE                             = $(CORE_TEST_DIR)/firmware
 VERI_FIRMWARE                        = ../../tests/core/firmware
 CUSTOM                               = $(CORE_TEST_DIR)/custom
-VERI_CUSTOM                          = ../../tests/core/custom
```

Comment:
This is required.  The VERI_* variables are here because Verilator cannot handle arbitrarily long pathnames.